### ISSUE TITLE
Dark theme 

### ISSUE THREAD

**Issue by Phoenix1025** [floccusaddon/floccus#1630]
### Describe the feature you'd like to request

Your app works and i love it.
Can you add dark theme to it bro?

### Describe the solution you'd like

It would be great if you can add changing theme from light to dark, and other stuff related to theming bro.

### Describe alternatives you've considered

Maybe you can also add accent colors, stuff. 
Thanks bro.

**Comment by marcelklehr** [comment]
There is already a dark mode. It's synced with your android theme settings.

**Comment by Phoenix1025** [comment]
> There is already a dark mode. It's synced with your android theme settings.

Thats what i thought, because i enabled dark mode on my phone. But some apps refuses to use it. So i use darkQ app to force apply dark mode into them. Unfortunately it doesnt work on this app.

**Comment by marcelklehr** [comment]
So, you have dark mode enabled in your android settings but floccus is still in light mode?

**Comment by Phoenix1025** [comment]
> So, you have dark mode enabled in your android settings but floccus is still in light mode?

Yes, thats right. There are other apps also. Like inshare  which also doesnt have theme settings to choose dark or light mode. But  darkQ can force it to dark mode

**Comment by Phoenix1025** [comment]
> > So, you have dark mode enabled in your android settings but floccus is still in light mode?
> 
> Yes, thats right. There are other apps also. Like inshare  which also doesnt have theme settings to choose dark or light mode. But  darkQ can force it to dark mode

Its in dark mode now that i reinstalled google drive bro.
